crates/holochain_types/src/dht_op/facts.rs

```diff
@@ -13,7 +13,7 @@ pub fn valid_dht_op(
     keystore: MetaLairClient,
     author: AgentPubKey,
     must_be_public: bool,
-) -> Facts<'static, DhtOp> {
```

Comment:
This will now point at a yanked crate, it's probably worth discussing going to `0.2.0-xxx` of this library to sync up with the fixes here